assertj-tests/assertj-integration-tests/assertj-core-tests/src/test/java/module-info.java.disabled

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Descriptor not yet in use due to mockito/mockito#3037.

Reply:
Solved with https://github.com/mockito/mockito/issues/3037#issuecomment-2664842234.